### ISSUE TITLE
Registered users + not updating information

### ISSUE THREAD

**Issue by tiimk** [yariplus/nodebb-plugin-minecraft-integration#94]
So the new update fixed nodebb crashing when players registered. However under the registered users tabs it doesn't update their player name it just shows undefined. I've relogged, refreshed, and restarted with no luck. Also not as important but the information such as rank and all of that doesn't get updated until a user relogs. Not sure if there is a way to fix that.

**Comment by yariplus** [comment]
Redid the Registered Users page so it now looks similar to the user's own page. It lists all the user's players and labels their primary player.

(I'll be redoing the commands, refresh/delete/make primary, later. I removed them for now since they don't work.)

**Comment by yariplus** [comment]
Hmm, I know about the rank not updating until relog, I think I should be able to send it with the registration message so that it updates immediately.

**Comment by tiimk** [comment]
Registered user page seems to work I like how you have it setup. Glad that you are actively developing this. The one last thing I've found is with the avatar style it seems they are all flat. I know cravatar.eu has a helmhead one and I attempted to change the config file to account for that but is seemed to break nodebb. Not sure if you'd be able to fix that so the style could be helmhead but that'd be great :)
